lib/api/holdings_api.rb

```diff
@@ -122,4 +121,15 @@ def validate_params(params:, required:)
       raise ArgumentError, argument_errors.join("; ")
     end
   end
+
+  # Should be used in any route that takes an organization as an argument,
+  # to check that it is either an existing organization or a mapto.
+  def organization_exists?(organization)
```

Comment:
Would it make sense for `HtOrganizations` to have a `has_organization?` method? Obviously we need to raise the `ArgumentError` here.